database/db.go

```diff
@@ -15,19 +15,24 @@ var (
 
 // DBConfig wraps config for connecting to a database.
 type DBConfig struct {
-	Address  string
-	User     string
-	Database string
-	Password string
+	Address       string
+	User          string
+	Database      string
+	Password      string
+	Logger        *log.Logger
```

Comment:
Should this just be made a Service Logger at this point? Do we fully standardize on it?

Reply:
This can be a service logger - its at the point now with the last updates Michael made to not have a logger per level and accept human readable log level that its ready for prime time/doesn't have any major usability issues I can see(take that with a grain of salt as I haven't used it in any of the code I've written)

Reply:
It's my proudest go code. That aside, switching between the two is fairly painless and none of the consumer code for the logger will have to change.